documentation/en/draft_modulators.md

```diff
@@ -0,0 +1,370 @@
+
+#### 2.3.2 Transverse waves
+In the transverse waves the particles vibrate perpendicularly to the propagation of the wave. Like the sea waves, waves in a string, and seismic waves.
+
+### 2.4 Periodicity
+
+#### 2.4.1 Periodic wave
+The local disturbance that originates this type of waves it's produced in repetitive cicles, e.g. a sine wave.
+
+#### 2.4.2 Aperiodic Non-periodic wave 
+
+The disturbance that originates them are isolated, or in case it repeats, the following disturbances have different characteristics. These isolated waves are known as pulse wave.
+
+
+# Chapter 2 Modulators and demodulators
+
+Information signals are rarely in an appropiate state for transmission, so that signals must be carry between the transmisor and the receiver on some medium for transmission.
+
+Modulation is the process of transforming the information from its original form to a more appropiate one for transmission. 
+
+Demodulation is the reverse process, turning the modulated wave to its original form. Modulation its done in the transmisor, specifically in the modulator, and the demodulation in the receiver side on a circuit called demodulator or detector.
+
+## 2.1 Carrier wave
+
+A carrier wave is an electric wave that can be altered in some of its parameters for the information signal (sound, image or data) to obtain a modulated signal and be transported on a communication channel.
+
+The use of a carrier wave helps solve many circuit, antenna, propagation and noise problems. That's why a practical antenna must have an approximate size to the lenght of the electromagnetic wave to be transmitted. If soundwaves diffused directly in the form of electromagnetic signals, the antenna would have the height of more than 1 Km. By using frequencies much more higher for the carrier, the size of the antenna its reduced drastically because higher frequencies have shorter wavelenght. 
+
+## 2.2 Modulated signal
+
+A modulated signal can be a signal of audio, video, or data. Any of these signals mixed with the carrier generate the modulated signal transmitted through the antenna. 
+
+
+<img src="imple_pic/modulator1.svg" alt="drawing" height="180" width="400" align="block"/>
+
+
+## 2.3 Why modulate a signal?
+
+For different reasons:
+
+**a**. If all users transmit to the same frequency of the original signal (modulated), it won't be possible to recognize the information contained in it, due to jamming between the signals transmitted from different users.
+
+**b**. There is more efficiency in the transmission at higher frequencies.
+
+**c**. It's possible to take advantage of the electromagnetic spectrum because it allows frequency multiplexing. 
+
+**d**. In case of wireless transmission, the antennas have more reasonable sizes. 
+
+In brief, modulation allows to take advantage of the communication channel, by making possible the transmission of more data simultaneously over the same channel and protect the data from possible interferences or noises.
```

Comment:
```suggestion
In brief, modulation allows to take advantage of the communication channel, by making possible the transmission of more data simultaneously over the same channel and protect the data from possible interference or noises.
```

Reply:
if `interference` is plural/singular also` noise` should be